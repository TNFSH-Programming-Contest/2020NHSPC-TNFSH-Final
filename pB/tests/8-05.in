568
-803.988 -218.608
-801.52 -227.489
-798.955 -236.341
-796.292 -245.164
-793.531 -253.958
-790.673 -262.72
-787.719 -271.45
-784.668 -280.147
-781.521 -288.809
-778.278 -297.437
-774.941 -306.028
-771.508 -314.581
-767.981 -323.096
-764.36 -331.571
-760.645 -340.006
-756.838 -348.4
-752.938 -356.75
-748.945 -365.057
-744.861 -373.319
-740.686 -381.536
-736.42 -389.706
-732.065 -397.828
-727.619 -405.902
-723.085 -413.926
-718.462 -421.899
-713.751 -429.82
-708.953 -437.69
-704.068 -445.505
-699.096 -453.266
-694.04 -460.971
-688.898 -468.62
-683.672 -476.212
-678.363 -483.746
-672.97 -491.22
-667.495 -498.634
-661.939 -505.987
-656.301 -513.278
-650.583 -520.507
-644.786 -527.672
-638.909 -534.772
-632.955 -541.806
-626.923 -548.775
-620.814 -555.676
-614.629 -562.509
-608.369 -569.274
-602.035 -575.969
-595.627 -582.593
-589.146 -589.146
-582.593 -595.627
-575.969 -602.035
-569.274 -608.369
-562.509 -614.629
-555.676 -620.814
-548.775 -626.923
-541.806 -632.955
-534.772 -638.909
-527.672 -644.786
-520.507 -650.583
-513.278 -656.301
-505.987 -661.939
-498.634 -667.495
-491.22 -672.97
-483.746 -678.363
-476.212 -683.672
-468.62 -688.898
-460.971 -694.04
-453.266 -699.096
-445.505 -704.068
-437.69 -708.953
-429.82 -713.751
-421.899 -718.462
-413.926 -723.085
-405.902 -727.619
-397.828 -732.065
-389.706 -736.42
-381.536 -740.686
-373.319 -744.861
-365.057 -748.945
-356.75 -752.938
-348.4 -756.838
-340.006 -760.645
-331.571 -764.36
-323.096 -767.981
-314.581 -771.508
-306.028 -774.941
-297.437 -778.278
-288.809 -781.521
-280.147 -784.668
-271.45 -787.719
-262.72 -790.673
-253.958 -793.531
-245.164 -796.292
-236.341 -798.955
-227.489 -801.52
-218.608 -803.988
-209.702 -806.357
-200.769 -808.627
-191.812 -810.798
-182.831 -812.871
-173.828 -814.843
-164.804 -816.716
-155.76 -818.489
-146.696 -820.162
-137.615 -821.735
-128.517 -823.207
-119.403 -824.578
-110.274 -825.848
-101.132 -827.018
-91.978 -828.086
-82.812 -829.052
-73.636 -829.918
-64.452 -830.682
-55.259 -831.344
-46.059 -831.904
-36.854 -832.363
-27.645 -832.719
-18.432 -832.974
-9.216 -833.127
-0.0 -833.178
9.216 -833.127
18.432 -832.974
27.645 -832.719
36.854 -832.363
46.059 -831.904
55.259 -831.344
64.452 -830.682
73.636 -829.918
82.812 -829.052
91.978 -828.086
101.132 -827.018
110.274 -825.848
119.403 -824.578
128.517 -823.207
137.615 -821.735
146.696 -820.162
155.76 -818.489
164.804 -816.716
173.828 -814.843
182.831 -812.871
191.812 -810.798
200.769 -808.627
209.702 -806.357
218.608 -803.988
227.489 -801.52
236.341 -798.955
245.164 -796.292
253.958 -793.531
262.72 -790.673
271.45 -787.719
280.147 -784.668
288.809 -781.521
297.437 -778.278
306.028 -774.941
314.581 -771.508
323.096 -767.981
331.571 -764.36
340.006 -760.645
348.4 -756.838
356.75 -752.938
365.057 -748.945
373.319 -744.861
381.536 -740.686
389.706 -736.42
397.828 -732.065
405.902 -727.619
413.926 -723.085
421.899 -718.462
429.82 -713.751
437.69 -708.953
445.505 -704.068
453.266 -699.096
460.971 -694.04
468.62 -688.898
476.212 -683.672
483.746 -678.363
491.22 -672.97
498.634 -667.495
505.987 -661.939
513.278 -656.301
520.507 -650.583
527.672 -644.786
534.772 -638.909
541.806 -632.955
548.775 -626.923
555.676 -620.814
562.509 -614.629
569.274 -608.369
575.969 -602.035
582.593 -595.627
589.146 -589.146
595.627 -582.593
602.035 -575.969
608.369 -569.274
614.629 -562.509
620.814 -555.676
626.923 -548.775
632.955 -541.806
638.909 -534.772
644.786 -527.672
650.583 -520.507
656.301 -513.278
661.939 -505.987
667.495 -498.634
672.97 -491.22
678.363 -483.746
683.672 -476.212
688.898 -468.62
694.04 -460.971
699.096 -453.266
704.068 -445.505
708.953 -437.69
713.751 -429.82
718.462 -421.899
723.085 -413.926
727.619 -405.902
732.065 -397.828
736.42 -389.706
740.686 -381.536
744.861 -373.319
748.945 -365.057
752.938 -356.75
756.838 -348.4
760.645 -340.006
764.36 -331.571
767.981 -323.096
771.508 -314.581
774.941 -306.028
778.278 -297.437
781.521 -288.809
784.668 -280.147
787.719 -271.45
790.673 -262.72
793.531 -253.958
796.292 -245.164
798.955 -236.341
801.52 -227.489
803.988 -218.608
806.357 -209.702
808.627 -200.769
810.798 -191.812
812.871 -182.831
814.843 -173.828
816.716 -164.804
818.489 -155.76
820.162 -146.696
821.735 -137.615
823.207 -128.517
824.578 -119.403
825.848 -110.274
827.018 -101.132
828.086 -91.978
829.052 -82.812
829.918 -73.636
830.682 -64.452
831.344 -55.259
831.904 -46.059
832.363 -36.854
832.719 -27.645
832.974 -18.432
833.127 -9.216
833.178 0.0
833.127 9.216
832.974 18.432
832.719 27.645
832.363 36.854
831.904 46.059
831.344 55.259
830.682 64.452
829.918 73.636
829.052 82.812
828.086 91.978
827.018 101.132
825.848 110.274
824.578 119.403
823.207 128.517
821.735 137.615
820.162 146.696
818.489 155.76
816.716 164.804
814.843 173.828
812.871 182.831
810.798 191.812
808.627 200.769
806.357 209.702
803.988 218.608
801.52 227.489
798.955 236.341
796.292 245.164
793.531 253.958
790.673 262.72
787.719 271.45
784.668 280.147
781.521 288.809
778.278 297.437
774.941 306.028
771.508 314.581
767.981 323.096
764.36 331.571
760.645 340.006
756.838 348.4
752.938 356.75
748.945 365.057
744.861 373.319
740.686 381.536
736.42 389.706
732.065 397.828
727.619 405.902
723.085 413.926
718.462 421.899
713.751 429.82
708.953 437.69
704.068 445.505
699.096 453.266
694.04 460.971
688.898 468.62
683.672 476.212
678.363 483.746
672.97 491.22
667.495 498.634
661.939 505.987
656.301 513.278
650.583 520.507
644.786 527.672
638.909 534.772
632.955 541.806
626.923 548.775
620.814 555.676
614.629 562.509
608.369 569.274
602.035 575.969
595.627 582.593
589.146 589.146
582.593 595.627
575.969 602.035
569.274 608.369
562.509 614.629
555.676 620.814
548.775 626.923
541.806 632.955
534.772 638.909
527.672 644.786
520.507 650.583
513.278 656.301
505.987 661.939
498.634 667.495
491.22 672.97
483.746 678.363
476.212 683.672
468.62 688.898
460.971 694.04
453.266 699.096
445.505 704.068
437.69 708.953
429.82 713.751
421.899 718.462
413.926 723.085
405.902 727.619
397.828 732.065
389.706 736.42
381.536 740.686
373.319 744.861
365.057 748.945
356.75 752.938
348.4 756.838
340.006 760.645
331.571 764.36
323.096 767.981
314.581 771.508
306.028 774.941
297.437 778.278
288.809 781.521
280.147 784.668
271.45 787.719
262.72 790.673
253.958 793.531
245.164 796.292
236.341 798.955
227.489 801.52
218.608 803.988
209.702 806.357
200.769 808.627
191.812 810.798
182.831 812.871
173.828 814.843
164.804 816.716
155.76 818.489
146.696 820.162
137.615 821.735
128.517 823.207
119.403 824.578
110.274 825.848
101.132 827.018
91.978 828.086
82.812 829.052
73.636 829.918
64.452 830.682
55.259 831.344
46.059 831.904
36.854 832.363
27.645 832.719
18.432 832.974
9.216 833.127
0.0 833.178
-9.216 833.127
-18.432 832.974
-27.645 832.719
-36.854 832.363
-46.059 831.904
-55.259 831.344
-64.452 830.682
-73.636 829.918
-82.812 829.052
-91.978 828.086
-101.132 827.018
-110.274 825.848
-119.403 824.578
-128.517 823.207
-137.615 821.735
-146.696 820.162
-155.76 818.489
-164.804 816.716
-173.828 814.843
-182.831 812.871
-191.812 810.798
-200.769 808.627
-209.702 806.357
-218.608 803.988
-227.489 801.52
-236.341 798.955
-245.164 796.292
-253.958 793.531
-262.72 790.673
-271.45 787.719
-280.147 784.668
-288.809 781.521
-297.437 778.278
-306.028 774.941
-314.581 771.508
-323.096 767.981
-331.571 764.36
-340.006 760.645
-348.4 756.838
-356.75 752.938
-365.057 748.945
-373.319 744.861
-381.536 740.686
-389.706 736.42
-397.828 732.065
-405.902 727.619
-413.926 723.085
-421.899 718.462
-429.82 713.751
-437.69 708.953
-445.505 704.068
-453.266 699.096
-460.971 694.04
-468.62 688.898
-476.212 683.672
-483.746 678.363
-491.22 672.97
-498.634 667.495
-505.987 661.939
-513.278 656.301
-520.507 650.583
-527.672 644.786
-534.772 638.909
-541.806 632.955
-548.775 626.923
-555.676 620.814
-562.509 614.629
-569.274 608.369
-575.969 602.035
-582.593 595.627
-589.146 589.146
-595.627 582.593
-602.035 575.969
-608.369 569.274
-614.629 562.509
-620.814 555.676
-626.923 548.775
-632.955 541.806
-638.909 534.772
-644.786 527.672
-650.583 520.507
-656.301 513.278
-661.939 505.987
-667.495 498.634
-672.97 491.22
-678.363 483.746
-683.672 476.212
-688.898 468.62
-694.04 460.971
-699.096 453.266
-704.068 445.505
-708.953 437.69
-713.751 429.82
-718.462 421.899
-723.085 413.926
-727.619 405.902
-732.065 397.828
-736.42 389.706
-740.686 381.536
-744.861 373.319
-748.945 365.057
-752.938 356.75
-756.838 348.4
-760.645 340.006
-764.36 331.571
-767.981 323.096
-771.508 314.581
-774.941 306.028
-778.278 297.437
-781.521 288.809
-784.668 280.147
-787.719 271.45
-790.673 262.72
-793.531 253.958
-796.292 245.164
-798.955 236.341
-801.52 227.489
-803.988 218.608
-806.357 209.702
-808.627 200.769
-810.798 191.812
-812.871 182.831
-814.843 173.828
-816.716 164.804
-818.489 155.76
-820.162 146.696
-821.735 137.615
-823.207 128.517
-824.578 119.403
-825.848 110.274
-827.018 101.132
-828.086 91.978
-829.052 82.812
-829.918 73.636
-830.682 64.452
-831.344 55.259
-831.904 46.059
-832.363 36.854
-832.719 27.645
-832.974 18.432
-833.127 9.216
-833.178 0.0
-833.127 -9.216
-832.974 -18.432
-832.719 -27.645
-832.363 -36.854
-831.904 -46.059
-831.344 -55.259
-830.682 -64.452
-829.918 -73.636
-829.052 -82.812
-828.086 -91.978
-827.018 -101.132
-825.848 -110.274
-824.578 -119.403
-823.207 -128.517
-821.735 -137.615
-820.162 -146.696
-818.489 -155.76
-816.716 -164.804
-814.843 -173.828
-812.871 -182.831
-810.798 -191.812
-808.627 -200.769
-806.357 -209.702
484.673 103.394
