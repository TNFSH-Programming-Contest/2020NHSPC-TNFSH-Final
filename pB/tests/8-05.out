5546776.736
